src/Form/OgMembershipDeleteForm.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Drupal\og\Form;
+
+use Drupal\Core\Entity\ContentEntityDeleteForm;
+
+/**
+ * Provides a form for deleting a node.
```

Comment:
@jludwig 
Not a node entity anymore, comment to be actualized.

Reply:
This became:
 * Provides a form for deleting an OG membership.